html/say-what-admin-addedit.php

```diff
@@ -15,11 +15,11 @@
 			<input type="text" name="say_what_orig_string" size="120" value="<?php echo esc_attr( $replacement->orig_string ) ?>"><br/>
 		</p>
 		<p>
-			<label for="say_what_domain"><?php _e( 'Text domain', 'say_what' ); ?></label> <a href="http://plugins.leewillis.co.uk/doc_post/adding-string-replacement/"><i class="dashicons dashicons-info">&nbsp;</i></a><br/>
+			<label for="say_what_domain"><?php _e( 'Text domain', 'say_what' ); ?></label> <a href="http://plugins.leewillis.co.uk/doc_post/adding-string-replacement/" target="_blank" tabindex="-1"><i class="dashicons dashicons-info">&nbsp;</i></a><br/>
```

Comment:
Adding a tabindex of -1 makes the link inaccessible to  people relying on assistive technologies (or who like keyboard navigation) so I wouldn't accept this as-is. Can you either remove that attribute of it was accidental, or if it was deliberate use an alternative approach. Thanks